tchannel/schemes/__init__.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+RAW = 'raw'
+JSON = 'json'
+THRIFT = 'thrift'
+DEFAULT = RAW
+
+DEFAULT_NAMES = (
+    RAW,
+    JSON,
+    THRIFT
+)
+
+from .raw import RawArgScheme  # noqa
+from .json import JsonArgScheme  # noqa
+from .thrift import ThriftArgScheme  # noqa
```

Comment:
What? You're doing intentional circular imports? Surely there's a better way? Move the constants to another schemes/foo.py and import that everywhere?